packages/app/src/app/pages/Dashboard/Components/UpgradeBanner/UpgradeBanner.tsx

```diff
@@ -0,0 +1,211 @@
+// above the call to action.
+const WRAP_WIDTH = 1332;
+
+type Feature = {
+  icon: IconNames;
+  label: string;
+};
+const FEATURES: Feature[] = [
+  {
+    icon: 'profile',
+    label: 'Up to 20 editors',
+  },
+  {
+    icon: 'npm',
+    label: 'Private NPM packages',
+  },
+  {
+    icon: 'sandbox',
+    label: 'Unlimited sandboxes',
+  },
+  {
+    icon: 'lock',
+    label: 'Advanced privacy settings',
+  },
+  {
+    icon: 'repository',
+    label: 'Unlimited repositories',
+  },
+  {
+    icon: 'server',
+    label: '6GB RAM, 12GB Disk, 4 vCPUs',
+  },
+];
+
+type UpgradeBannerProps = {
+  isAdmin: boolean;
+  teamId: string;
+};
+export const UpgradeBanner: React.FC<UpgradeBannerProps> = ({
+  isAdmin,
+  teamId,
+}) => {
+  const [checkout, createCheckout] = useCreateCheckout();
+  const [isBannerDismissed, dismissBanner] = useDismissible(
+    'DASHBOARD_RECENT_UPGRADE'
+  );
+
+  if (isBannerDismissed) {
+    return null;
+  }
+
+  const checkoutBtnDisabled = !isAdmin || checkout.status === 'loading';
+
+  return (
+    <Banner
+      onDismiss={() => {
+        track('Banner - Upgrade Now - Dismiss', {
+          codesandbox: 'V1',
+          event_source: 'UI',
+        });
+
+        dismissBanner();
+      }}
+    >
+      <Element css={{ overflow: 'hidden' }}>
+        <StyledTitle color="#EDFFA5" weight="600" block>
+          Upgrade to <span css={{ textTransform: 'uppercase' }}>pro</span>
+        </StyledTitle>
+        <Stack
+          css={{
+            flexWrap: 'wrap',
+          }}
+          justify="space-between"
+        >
+          <Stack direction="vertical" justify="space-between">
+            <StyledTitle block>
+              Enjoy the full CodeSandbox experience.
+            </StyledTitle>
+            <Stack
+              align="center"
+              css={{
+                [`@media screen and (max-width: ${WRAP_WIDTH}px)`]: {
+                  marginTop: '24px',
+                },
+              }}
+              gap={6}
+            >
+              {isAdmin ? (
+                <>
+                  <Button
+                    onClick={() => {
+                      if (checkoutBtnDisabled) {
+                        return;
+                      }
+
+                      track('Banner - Upgrade now - Checkout', {
+                        codesandbox: 'V1',
+                        event_source: 'UI',
+                      });
+
+                      createCheckout({
+                        team_id: teamId,
+                        recurring_interval: 'month',
+                        success_path: dashboard.recent(teamId),
+                        cancel_path: dashboard.recent(teamId),
+                      });
+                    }}
+                    loading={checkout.status === 'loading'}
+                    disabled={checkoutBtnDisabled}
+                    type="button"
+                    autoWidth
+                  >
+                    Upgrade now
```

Comment:
Should we show here `Start Trial` vs `Upgrade now` based on the conditional if the team is eligible for a trial? I believe @tristandubbeld is abstracting that in a hook from the conditional in the sidebar bottom message.